plugins/woocommerce/src/Internal/Traits/SourceAttributionMeta.php

```diff
@@ -230,19 +230,39 @@ private function get_hpos_order_object( $post_or_order ) {
 		return $post_or_order;
 	}
 
+
 	/**
-	 * Map posted values to meta values.
+	 * Map posted, prefixed values to fields.
+	 * Used for the classic forms.
 	 *
-	 * @param array $raw_values The raw values from the submitted data.
+	 * @param array $raw_values The raw values from the POST form.
 	 *
 	 * @return array
 	 */
-	private function get_source_values( array $raw_values = array() ): array {
+	private function get_unprefixed_fields( array $raw_values = array() ): array {
```

Comment:
I was not sure whether it's better to keep it here or move to `SourceAttributionController` as it's only used there. On the other hand, all the other `*prefixed_` related methods are here.